sh_scrapy/compat.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import unicode_literals
```

Comment:
AFAIK there's a general rule to avoid `unicode_literals` when porting Py2 to Py3 - e.g. see http://python-future.org/unicode_literals.html. Have you double checked we won't hit any of those?